### PR TITLE
Moving the Stats service from storagefacility to thingaggregate.

### PR DESCRIPTION
**Pull request by kpmeen**
It is a more appropriate place for it. And it will allow us to include more object related stats later on.

### CODE REVIEW

File: service_thing_aggregate/app/repositories/dao/ObjectDao.scala [hunk 1]
```diff
@@ -290,15 +287,15 @@ class ObjectDao @Inject() (
...
+    val locObjQuery = locObjTable.filter { lo =>
       lo.museumId === mid &&
         lo.currentLocationId === nodeId
     }
 
     // Filter on collection access if the user doesn't have GodMode
     val objQuery = {
-      if (currUsr.hasGodMode) objects
-      else objects.filter { o =>
+      if (currUsr.hasGodMode) objTable
```
Comment (jarlah) [inline service_thing_aggregate/app/repositories/dao/ObjectDao.scala:297]: her er codacy ut å sykler totalt ass

File: service_thing_aggregate/app/services/StatsService.scala [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  def nodeStats(
+    mid: MuseumId,
+    nodeId: StorageNodeDatabaseId
+  )(implicit currUsr: AuthenticatedUser): Future[MusitResult[Option[NodeStats]]] = {
+    MusitResultT(nodeDao.getPathById(mid, nodeId)).flatMap { maybeNode =>
+      maybeNode.map { node =>
+        val totalF = MusitResultT(statsDao.numObjectsInPath(node._2))
+        val directF = MusitResultT(statsDao.numObjectsInNode(node._1))
+        val nodeCountF = MusitResultT(statsDao.numChildren(node._1))
+        //        val perCol = MusitResultT(statsDao.numObjectsInNodeByCollection(
```
Comment (jarlah) [inline service_thing_aggregate/app/services/StatsService.scala]: remove?

**Review by jarlah — CHANGES_REQUESTED** [review]
seems to be ok. just a minor thing about commented code block that can be removed

File: service_thing_aggregate/app/repositories/dao/ObjectDao.scala (hunk 1 above)
Comment (kpmeen) [reply service_thing_aggregate/app/repositories/dao/ObjectDao.scala:297]: Jepp...pleier å gå inn for og eksplisitt ignorere det.

File: service_thing_aggregate/app/services/StatsService.scala (hunk 2 above)
Comment (kpmeen) [reply service_thing_aggregate/app/services/StatsService.scala]: no...that's where the counter per collection will come...very soon

**Review by jarlah — APPROVED** [review]

**Pull request merged**